utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/App.java

```diff
@@ -158,6 +162,19 @@ String postWafXml(@RequestBody XmlObject object) {
         return object.toString();
     }
 
+    @GetMapping(value = "/session/new")
```

Comment:
Could you document these new endpoints in `docs/weblog`? 🙏

Reply:
Yep, completely forgot that we have that documentation, updated, thanks!